ui-v2/app/services/repository.js

```diff
@@ -32,7 +32,7 @@ export default Service.extend({
             }
           }
           const date = get(item, 'SyncTime');
-          if (typeof date !== 'undefined' && date != meta.date) {
+          if (!item.isDeleted && typeof date !== 'undefined' && date != meta.date) {
```

Comment:
A blocking query can resolve with a recordset with a deleted item (that it tries to reconcile out) whilst ember-data still hasn't completely deleted/unloaded the item. This prevents ember erroring when that happens.